jbmc/src/jbmc/jbmc_parse_options.h

```diff
@@ -48,7 +48,6 @@ class optionst;
   "(object-bits):" \
   "(classpath):(cp):(main-class):" \
   "(no-assertions)(no-assumptions)" \
-  "(no-built-in-assertions)" \
```

Comment:
Oh, this should be removed from `cbmc_parse_options.h` as well. Would you mind adding such a commit?

Reply:
Why? This option is still present in goto_check. Shall I remove it entirely?